app/views/login.scala.html

```diff
@@ -0,0 +1,83 @@
+<!--
+    Simple Bootstrap login page
+    Source: https://github.com/HuyLafa/MathTutor
+-->
+
+@main("Log In") {
+<div class="container">
+    <div class="row">
+        <div class="col-md-6 col-md-offset-3">
+            <div class="panel panel-login">
+                <div class="panel-heading">
+                    <div class="row">
+                        <div class="col-xs-6">
+                            <a href="#" class="active" id="login-form-link">Login</a>
+                        </div>
+                        <div class="col-xs-6">
+                            <a href="#" id="register-form-link">Register</a>
+                        </div>
+                    </div>
+                    <hr>
+                </div>
+                <div class="panel-body">
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <form id="login-form" action="http://phpoll.com/login/process" method="post" role="form" style="display: block;">
```

Comment:
On second thought, let's keep this as simple as possible for now -- maybe just have people enter a nickname?